src/components/Login/Login.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import Layout from "components/Layout/Layout";
+
+export default function Login() {
+  return (
+    <Layout sidebar={false}>
```

Comment:
IIRC we want to render the application in a disconnected state and then have a login modal/dialogue rendered instead of going a full page like this.

Reply:
OK, can be refactored in due course.